docs/lab/overview/testing-modalities.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: "Testing Modalities"
+---
+
+By integrating with our API, you will have access to all testing modalities.
+
+Three types are currently supported:
+
+- At-home test kit.
+- At-home phlebotomy visit.
+- Walk-in test visit.
+
+## At-home test kits
+
+This involves sending a kit to a patients home that allows the patient to collect their
+sample specimen and send back to our lab networks in the mail.
+
+At home testing kits are comprised of a number of components such as:
+
+ <div className="flex flex-row">
+  <div>
+
+- Mailer Box
+- Blade Lancets
+- Plasters
+- Gauze
+- ADX Card
+- Blood collection card
+- Saliva Tubes
+- Collection Supplies
+- Instruction Booklet
+
+Additional components can be included in the test kit box, to fully customize the kit, just contact us support@tryvital.io.
+
+  </div>
+  <img src="/img/wyndly.png" className="w-1/2 h-1/2" />
+</div>
+
+## At-home phlebotomy
+
+At-home phlebotomy requires mobile phlebotomists visiting a patient at their home and carrying out the phlebotomy draw. The sample is then delivered to the lab by the phlebotomist.
```

Comment:
The mobile word feels odd there, but maybe it's right nomenclature here?

I would have worded it as:

```
At-home phlebotomy requires a phlebotomist to visit a patient's home and carrying out the phlebotomy draw. (...)
```